app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -2600,11 +2600,9 @@ class ChatActivity :
             chatMessage.activeUser = conversationUser
 
             adapter?.let {
-                chatMessage.isGrouped = (
-                    it.isPreviousSameAuthor(chatMessage.actorId, -1) &&
-                        it.getSameAuthorLastMessagesCount(chatMessage.actorId) %
-                        GROUPED_MESSAGES_SAME_AUTHOR_THRESHOLD > 0
-                    )
+                val previousChatMessage = adapter!!.items[1].item as ChatMessage
```

Comment:
As said, there must be checks if the previousChatMessage is a ChatMessage.
To reproduce this crashes when the check is not in place, write a message from webuser to androiduser in a chat that does not yet have any message for this day.